src/util/inherit.py

```diff
@@ -63,7 +63,7 @@ def _get_base_version(cr):
         state, version = cr.fetchone()
         if state != "to upgrade":
             major = ".".join(version.split(".")[:2])
-            _logger.warning(
+            _logger.info(
```

Comment:
We could make almost a warn in CI.
```suggestion
            _logger.log(
                util.NEARLYWARN if util.on_CI() else logging.WARNING,
```